web/src/components/grouped-score-badge.tsx

```diff
@@ -43,9 +59,13 @@ const ScoresOfGroup = (props: {
 export const GroupedScoreBadges = ({
```

Comment:
do we still need this when moving to single-score aggregates that are not grouped together?